hal/src/msom/pinmap_defines.cpp

```diff
@@ -69,7 +69,7 @@ static hal_pin_info_t pinmap[TOTAL_PINS] = {
 /* BGVINT        - 40 */ { RTL_PORT_B,     31, PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
 /* BGDTR         - 41 */ { RTL_PORT_B,     30, PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
 /* GNSS_ANT_PWR  - 42 */ { RTL_PORT_A,      2, PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
-/* VBAT_MEAS/A7  - 43 */ { RTL_PORT_NONE,   0, PIN_MODE_NONE, PF_NONE, 7,                PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
+/* VBAT_MEAS/A7  - 43 */ { RTL_PORT_A,     20, PIN_MODE_NONE, PF_NONE, 7,                PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
```

Comment:
Should we go as far as to remove it from the array of pins here? (and adjust all the dependend constants etc)